sympy/physics/control/lti.py

```diff
@@ -24,19 +34,48 @@ def var(self):
         return self._var
 
     def __add__(self, other):
-        pass
+        if not self.var == other.var:
+            raise ValueError("Last argument for both Transfer Functions must be same.")
+        p = self.num * other.den + other.num * self.den
+        q = self.den * other.den
+        return TransferFunction(expand(p), expand(q), self.var)
 
     def __sub__(self, other):
-        pass
+        if not self.var == other.var:
+            raise ValueError("Last argument for both Transfer Functions must be same.")
+        p = self.num * other.den - other.num * self.den
+        q = self.den * other.den
+        return TransferFunction(expand(p), expand(q), self.var)
 
     def __mul__(self, other):
-        pass
+        if not self.var == other.var:
+            raise ValueError("Last argument for both Transfer Functions must be same.")
+        p = self.num * other.num
+        q = self.den * other.den
+        return TransferFunction(expand(p), expand(q), self.var)
 
     def __div__(self, other):
-        pass
+        if not self.var == other.var:
+            raise ValueError("Last argument for both Transfer Functions must be same.")
+        p = self.num * other.den
+        q = self.den * other.num
+        return TransferFunction(expand(p), expand(q), self.var)
+
+    __truediv__ = __div__
 
     def __pow__(self, p):
-        pass
+        p = sympify(p)
+        if not isinstance(p, Integer):
+            raise ValueError("Exponent must be an Integer.")
+        if p == 0:
+            return TransferFunction(1, 1, self.var)
+        if p < 0:
+            p = abs(p)
+            num_, den_ = expand(self.den**p), expand(self.num**p)
+            return TransferFunction(num_, den_, self.var)
+        if p > 0:
+            num_, den_ = expand(self.num**p), expand(self.den**p)
+            return TransferFunction(num_, den_, self.var)
```

Comment:
We can just do 
```python
else:
    p = abs(p)
    num_, den_ = expand(self.den**p), expand(self.num**p)
    return TransferFunction(num_, den_, self.var)   
```

`abs(p)` will return a positive number if p is negative, otherwise, it will remain positive.

Reply:
`num_` and `den_` are not the same here. When `p` is negative, ratio will be inverted.

Reply:
Oh yeah. Then you can do
```python
else:
    p = abs(p)
    num_, den_ = expand(self.den**p), expand(self.num**p)
    if p<0:
        num_, den_ = den_, num_
    return TransferFunction(num_, den_, self.var)   
```

Reply:
p will always be positive after using `abs`.

Reply:
Oh sorry, I missed that. Let me have another look at it.